website-frontend/src/components/NavBar.js

```diff
@@ -1,22 +1,36 @@
 // @nhomnhomm Complete this
+import {Link} from 'react-router-dom'
+import logo from '../images/logo.png'
+import './NavBar.scss'
 
-import { Link } from "react-router-dom"
-
-const NavBtn = ({path, text}) => {
+const NavLink = ({path, text}) => {
   return (
-    <Link to={ path }>
-      <button>
-        { text }
-      </button>
+    <Link to = {path} className='nav-item nav-link nav-btn text-primary px-3 py-3'>
+      { text }
```

Comment:
I just updated this a bit (https://github.com/npnkhoi/stem-club-website/pull/8/commits/c42dbaa02db803f2bd8f3b0e9a0a2a3c84e3dd18) to achieve fancier hover style. Sorry this is not express in the Figma. What do you think about this new style?

![nav-hover](https://user-images.githubusercontent.com/11204020/159167801-0bf601c5-b90e-4bb2-badc-75dbc73ea7ac.gif)

Reply:
I personally think this functions well for its purpose; however, I do find it looking a bit too blocky (in pairing with the cover image with its rounded radius). I try adding the button radius (both with and without button margin) for this too but it does not look much distinguishedly better. Therefore, though it is not the best design, it is still a great option which I think we should keep if there is no better suggestion at hand.

![image](https://user-images.githubusercontent.com/87293327/159176877-72c70523-65a7-4f74-a6b0-5ef5a4f4dd24.png)